ci/jenkins/test-vmc.sh

```diff
@@ -273,11 +285,12 @@ function deliver_antrea {
         else
             VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" make && break
         fi
+        VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" make flow-aggregator-ubuntu && break
     done
     cd ci/jenkins
 
     if [ "$?" -ne "0" ]; then
-        echo "=== Antrea Image build failed ==="
+        echo "=== Antrea Image and Flow Aggregator Image build failed ==="
         exit 1
     fi
```

Comment:
I don't understand this code so I think we need @lzhecheng to review. Specifically I am unsure why we try to build the images twice. Also it seems that the `[ "$?" -ne "0" ]` test would apply to the `cd` command (last command run), not the image build.

Finally, maybe the error message should be "=== Antrea Image or Flow Aggregator Image build failed ==="

Reply:
could we just compile once as a workaround? I added some code for that. Earlier code did not work. Will see if new code succeeds.